index.html

```diff
@@ -18,7 +18,7 @@
                </nav>
             </Header>
         <div class="P-Text">
```

Comment:
- [ ] While trying to remove the extra spacing from the right of your web page, you ended up adding the spacing to the left. I think you can make proper use of Flexbox to fix this issue and set the entire view width to 100%

![image](https://github.com/Tshepo45/Mobile-First/assets/63473584/92a3267a-3625-4332-8d73-a71b3a8c62d1)